lighthouse-core/report/styles/report.css

```diff
@@ -363,23 +364,76 @@ body {
   border: none;
 }
 
-.report-body__header {
-  height: var(--report-header-height);
-  border-bottom: 1px solid #EBEBEB;
+.report-body__fixed-header-container {
   background: #FAFAFA;
   margin-left: var(--report-menu-width);
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: flex-start;
-  padding: 0 var(--heading-line-height);
   position: fixed;
   will-change: transform;
   z-index: 1;
   max-width: calc( var(--report-width) - var(--report-menu-width));
   width: calc(100vw - var(--report-menu-width));
 }
 
+.report-body__header {
+  height: var(--report-header-height);
+  border-bottom: 1px solid #EBEBEB;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0 var(--heading-line-height) 0 0;
+}
+
+.report-body__header-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg fill="black" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+  width: 24px;
+  height: 24px;
+  border: none;
+  flex: 0 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  background-color: transparent;
+  margin-left: 8px;
+}
+
+.report-body__fixed-header-container.expanded .report-body__header-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg fill="black" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+}
+
+.report-body__header-dropdown {
```

Comment:
I think it'd be better if the header just grew in side when you expand this.

So we could use a ▶ and it goes to ▼ when you click on it. `cursor: pointer` on the icon so its clearly actionable.